Demo/src/main/java/com/paypal/android/ui/paypal/StartPayPalNativeCheckoutForm.kt

```diff
@@ -0,0 +1,41 @@
+package com.paypal.android.ui.paypal
+
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.Spacer
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.layout.size
+import androidx.compose.material3.MaterialTheme
+import androidx.compose.material3.OutlinedCard
+import androidx.compose.material3.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.res.stringResource
+import androidx.compose.ui.unit.dp
+import com.paypal.android.R
+import com.paypal.android.ui.WireframeButton
+
+@Composable
+fun StartPayPalNativeCheckoutForm(
+    isLoading: Boolean,
+    onSubmit: () -> Unit
+) {
+
+    OutlinedCard(modifier = Modifier.fillMaxWidth()) {
+        Column(modifier = Modifier.padding(8.dp)) {
+            Text(
+                text = "Launch PayPal Web Checkout",
```

Comment:
Should this say native?

Reply:
Yeah 🙏 that's copy 🍝 at work haha. I'll fixup.